### ISSUE TITLE
Doesn't open a menu

### ISSUE THREAD

**Issue by Tanker348** [BlueSnoopT/Cunthook#2]
Doesn't open a menu when pressing insert, I've updated the cheat and multi restarts and it doesn't work still

**Comment by BlueSnoopT** [comment]
What OS are you attempting to use this on

**Comment by Tanker348** [comment]
> What OS are you attempting to use this on, could you also provide logs via **tmp/**? Are you also running it with no changes to Cmakelists.txt?

Oh sorry, it's fixed now, I had to go to fullscreen to fix that,
also can you fix crash? it's quite annoying and fixing could be a good update

**Comment by BlueSnoopT** [comment]
What crash?

**Comment by Tanker348** [comment]
> What crash?

Sometimes the game crashes, I can send the video if you need proof

**Comment by BlueSnoopT** [comment]
Just randomly?

**Comment by Tanker348** [comment]
> sequence

Randomly

**Comment by BlueSnoopT** [comment]
Hmm, send a log of one of those sessions where you crashed and I'll look into it

**Comment by Tanker348** [comment]
> Hmm, send a log of one of those sessions where you crashed and I'll look into it

how do I get the logs?

**Comment by BlueSnoopT** [comment]
Go to tmp/ via your root directory and find the .log file by most recently created date.

**Comment by Tanker348** [comment]
> Go to tmp/ via your root directory and find the .log file by most recently created date.

ok I found it the log
[14:14:14] Writing successful
[14:14:14] Pathing: Nav File location: /home/waterpark621/.steam/debian-installation/steamapps/common/Team Fortress 2/tf/maps/ctf_turbine.nav
[14:14:14] Init Glow...
[14:14:14] Init done!
[14:14:19] Found and hooked Minigun/Flamethrower!
[14:14:19] Hooking vtable 0xcee074a8 with 650 methods
[14:14:19] Hooking method 464 of vtable 0xcee074a8, replacing 0xce386450 with 0xc7d9abe0
[14:17:21] Manipulating KS 1
[14:17:21] Adjusting 0 -> 1
[14:18:14] Manipulating KS 2
[14:18:14] Adjusting 1 -> 2
[14:18:16] Adjusting 2 -> 0
[14:18:37] Manipulating KS 1
[14:18:37] Adjusting 0 -> 1
[14:19:46] Manipulating KS 2
[14:19:46] Adjusting 1 -> 2
[14:20:39] Manipulating KS 3
[14:20:39] Adjusting 2 -> 3
[14:21:21] Manipulating KS 4
[14:21:21] Adjusting 3 -> 4

**Comment by BlueSnoopT** [comment]
Alright thanks, although I'm pretty sure these random crashes are present in cathook as well. I'll look into it.

**Comment by Tanker348** [comment]
> Alright thanks, although I'm pretty sure these random crashes are present in cathook as well. I'll look into it.

also there is problem with cuntbot, the localhost website doesn't work even if you had ./start on

**Comment by BlueSnoopT** [comment]
Yes that is already an open issue with @russellcrowe-xpaha who has the same issue.
